package.json

```diff
@@ -109,6 +109,7 @@
 		"deep-freeze": "0.0.1",
 		"equivalent-key-map": "0.2.2",
 		"esbuild": "0.25.10",
+		"esbuild-esm-loader": "0.3.3",
```

Comment:
This is a [package](https://www.npmjs.com/package/esbuild-esm-loader) that I maintain and have used in a number of other projects. It itself is pretty light-weight and delegates most of the underlying work to [Node's module customization hooks](https://nodejs.org/docs/latest/api/module.html#customization-hooks) and ESBuild.

Reply:
Why is it added to the top level package instead of the theme package where it's used?

Reply:
I'm considering it a `devDependencies` and as far as I recall we define all of those in the root `package.json`?

Reply:
Updated in d02d9e072e (allow devDependencies) and 72ad4d18e6 (shift to package devDependency).